tests/integration/security/util/connection/checker.go

```diff
@@ -42,6 +47,13 @@ func (c *Checker) Check() error {
 			return fmt.Errorf("%s to %s:%s using %s: expected success but failed: %v",
 				c.From.Config().Service, c.Options.Target.Config().Service, c.Options.PortName, c.Options.Scheme, err)
 		}
+		// TODO: check why grpc can not reach all clusters
+		if len(c.DestClusters) > 1 && c.Options.Scheme != scheme.GRPC {
```

Comment:
curious, what are you seeing? a "didn't reach all clusters" or did you see failures? I can see why this would fail in plaintext mode - we need TLS for snirouting cross-network. If it's only failing in plaintext, you could do `CheckReachedClusters(c.DestClusters.ByNetwork()[c.From.Config().Cluster.NetworkName()])

Reply:
Hi Steven, you may check the last failed job at https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/27896/integ-security-multicluster-tests_istio/66/build-log.txt. It missed one cluster in the case of GRPC.

Reply:
Hmm... maybe bump the call count up, can be done in a follow up.